icons/play.svg

```diff
@@ -9,5 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <polygon points="5 3 19 12 5 21 5 3" />
+  <polygon points="7 3 21 12 7 21 7 3" />
```

Comment:
```suggestion
  <path d="m6 3 15 9-15 9z" />
```

I think moving it 2px to the right would make it appear even more off centre in most other standard use cases (e.g. on square buttons), but maybe 1px to the right and a 1px increase in width to make the triangle more equilateral would do the trick without drastically messing up vertical alignment to `square`:
![image](https://github.com/lucide-icons/lucide/assets/17746067/1af4016e-9a02-4974-9f36-80a910147153)
(current vs. 1px to the right vs. 2px to the right vs. with proposed changes)


![image](https://github.com/lucide-icons/lucide/assets/17746067/d3b75b2b-8d14-4660-b1ed-b6040965959e)
<img width="188" alt="image" src="https://github.com/lucide-icons/lucide/assets/17746067/8e98ebd5-5109-4aeb-be88-60fa190671f3">
(current vs. @bernaferrari vs. @karsa-mistmere)

Reply:
> Middle one looks best to me, pure gut feel tho.

Which middle one? 😄

Reply:
<img width="177" alt="image" src="https://github.com/lucide-icons/lucide/assets/25524993/bfde2635-2940-421c-93bf-8cddcc565ec2">
The middle one of those.

Reply:
The middle one looks the best to me as well.
@karsa-mistmere Is the middle one the suggestion you proposed?

Reply:
Nah, the middle one is simply 1px to the right, my proposal is the rightmost one:

![image](https://github.com/lucide-icons/lucide/assets/17746067/faa54787-721e-4826-8286-857f0c0463d4)
<img width="270" alt="image" src="https://github.com/lucide-icons/lucide/assets/17746067/96b8a1e9-f298-462b-bf55-595f4d07aadb">

Reply:
The second one and the most right one look both good to me 😅

Reply:
Agreed, either of these, I'd also prefer an equilateral(ish) play icon, but it's not a must.

Reply:
I vote for the second one.

Reply:
@karsa-mistmere Can provide the code suggestion for second one?

Reply:
I've updated the PR.